### ISSUE TITLE
make -j 4 doesn't work properly

### ISSUE THREAD

**Issue by mikrosk** [mikrosk/lib#19]
I rarely use the `-j` switch but somehow I had the feeling that at least gemlib and mintlib are parallel-builds friendly? However that doesn't seem to be the case, only the following folders are created: .lib020, .lib16, .lib, .lib02016

**Comment by th-otto** [comment]
Works as expected for me, and creates all 6 folders.

I recently update my system, but i use make -j8 all the time, especially with gemlib, where it makes a big difference in compilation times, and it always worked.

mintlib should work too. Only project which sometimes seem to cause trouble is the freemint kernel.

**Comment by mikrosk** [comment]
That's odd. I refuse to believe that it is caused by some WSL difference, most likely my build is just a bit slower due to virtualization.

Is there something I could check? I started with clean source tree.

**Comment by th-otto** [comment]
Are you using the linux version of gcc-m68k-atari-mint-gcc on WSL? Maybe you can also try the mingw version, available on my site. Maybe you need MSYS2 for a bash.

**Comment by mikrosk** [comment]
> Are you using the linux version of gcc-m68k-atari-mint-gcc on WSL

Indeed.

I'll test it in other environments, I vaguely remember trying `-j` at home and it worked.

**Comment by th-otto** [comment]
Today i had a similar issue.  make -j8 did not fail, but apparently did not rebuild anything. Running it again solved it.

Such issues are a pain. They are hard to track, because they depend on the order on how the commands are run.

**Comment by mikrosk** [comment]
Yeah, for that reason I rarely use the `-j` switch - our projects are small and usually don't require full recompilation.

**Comment by th-otto** [comment]
But for gemlib that isn't the case. Its not large, but there are a lot of small files, and make -j makes a big difference. And any change to mt_gem.h will trigger a full recompilation.

**Comment by mikrosk** [comment]
Fixed by commit d4b3772.
